src/styles/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  justify-content: center;
+
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &:hover &__title {
+    color: #34568b;
+  }
+
+  &__descr {
+    margin: 0 0 16px;
+    color: #616070;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__rate {
+    height: 16px;
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+    margin: 0 0 24px;
+  }
+
+  &__review {
+    text-align: right;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    height: 18px;
+    align-items: center;
+    justify-content: space-between;
+    margin: 0 0 16px;
+  }
+
+  &__text {
+    color: #616070;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__value {
+    text-align: right;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__btn {
+    color: #fff;
+    background-color: #00acdc;
+    width: 100%;
+    height: 40px;
+    font-size: 14px;
+    line-height: 40px;
+    font-weight: 700;
+    text-decoration: none;
+    text-align: center;
+    text-transform: uppercase;
+    border: 1px solid #00acdc;
```

Comment:
if u use same color more than twice, it should be a variable
also, u can put all colors in variables